TELEMETRY.md

```diff
@@ -0,0 +1,94 @@
+# Conjur Telemetry
+Conjur provides a configurable telemetry feature built on [Prometheus](https://prometheus.io/), which is the preferred open source monitoring tool for Cloud Native applications. When enabled, it will capture performance and usage metrics of the running Conjur instance. These metrics are exposed via a REST endpoint (/metrics) where Prometheus can scrape the data and archive it as a queryable time series. This increases the observability of a running Conjur instance and allows for easy integration with popular visualization and monitoring tools.
```

Comment:
nit: We usually wrap lines in MD files around 80 (?) characters. I think it's so:
- If the MD isn't being rendered, we still have some enforced formatting
- If the file is updated, the diffs aren't entire paragraphs, just a line's worth